File: 0136-single-number/solution.java
class Solution {
    public int singleNumber(int[] a) {
        Arrays.sort(a);
        int n=a.length;
        for(int i=0;i<a.length-1;i=i+2){
            if(a[i]!=a[i+1]) return a[i];
        }
        return a[n-1];
    }
}

File: 0136-single-number/solution.c
int singleNumber(int* a, int n) {
    int c=0;
    for(int i=0;i<n;i++){
        c=0;
         for(int j=1;j<n;j++){
            if(a[i]==a[(i+j)%n]){ ++c; break;}
         }
         if(c==0){
            return a[i];
         }
    }
    return 0;
}
